src/main/webapp/script.js

```diff
@@ -11,81 +11,74 @@
-'{ "q":"What would you choose for breakfast?" , "options":["acai bowl", "waffles", "crossaints", "huevos rancheros", "bagels"] },' +
-'{ "q":"Pick a TV show" , "options":["Emily in Paris", "Doctor Who", "Brooklyn 99", "Itaewon Class"] } ]}';
-var obj = JSON.parse(jsonPlacehold);
-var objIndex = 0;
-var responses = [];
-
-function onLoad(){
-    const quiz = document.getElementById('quiz');
-    quiz.appendChild(createParElement(obj.questions[0].q));
-    for (a in obj.questions[0].options){
-        // console.log(obj.questions[0].options[a])
-        var button = createButton(obj.questions[0].options[a])
-        button.addEventListener("click", function(){onClick(this);
-            });
+function onLoad() {
+    jsonPlacehold = '{ "questions" : [' +
+        '{ "q":"How would you spend an afternoon?" , "options":["hiking mountains", "shopping crafts", "trying local eats", "visiting a museum"]},' +
+        '{ "q":"What would you choose for breakfast?" , "options":["acai bowl", "waffles", "crossaints", "huevos rancheros", "bagels"] },' +
+        '{ "q":"Pick a TV show" , "options":["Emily in Paris", "Doctor Who", "Brooklyn 99", "Itaewon Class"] } ]}';
+    quizObject = JSON.parse(jsonPlacehold);
+    const quiz = document.getElementById(QUIZ_ELEMENT_ID);
+    quiz.appendChild(createParagraphElement(quizObject.questions[0].q));
+    for (let a of quizObject.questions[0].options) {
+        let button = createButton(a);
+        button.addEventListener("click", function() {
+            onClick(this);
+        });
         quiz.appendChild(button);
     }
 
 }
 
-function onClick(elm){
-    responses.push(elm.value);    
-    objIndex = objIndex+1;
-    // console.log(objIndex);
-    clearElm("quiz");
-    const quiz = document.getElementById('quiz');
-    quiz.appendChild(createParElement(obj.questions[objIndex].q));
-    for (a in obj.questions[objIndex].options){
-        // console.log(obj.questions[0].options[a])
-        var button = createButton(obj.questions[objIndex].options[a]);
-        if (objIndex==obj.questions.length-1){
-            button.addEventListener("click", function(){saveMatch(this);
+function onClick(elm) {
+    responses.push(elm.value);
+    quizIndex = quizIndex + 1;
+    clearElm(QUIZ_ELEMENT_ID);
+    const quiz = document.getElementById(QUIZ_ELEMENT_ID);
+    quiz.appendChild(createParagraphElement(quizObject.questions[quizIndex].q));
+    for (let a of quizObject.questions[quizIndex].options) {
+        let button = createButton(a);
+        if (quizIndex == quizObject.questions.length - 1) {
+            button.addEventListener("click", function() {
+                saveMatch(this);
             });
-        }
-        else {
-            button.addEventListener("click", function(){onClick(this);
+        } else {
+            button.addEventListener("click", function() {
+                onClick(this);
             });
         }
         quiz.appendChild(button);
     }
```

Comment:
This code could be pulled into a helper function and reused in onLoad. There's a case where there's only one question, in which case the 'if (quizIndex == quizObject.questions.length - 1)' would be necessary in the onLoad function.